portfolio/src/main/webapp/script.js

```diff
@@ -70,3 +70,14 @@ function contact() {
 		</ul>`; 
 	console.log('Contact!'); 
 }
+
+function getComments() {
+	fetch('/data').then(response => response.json()).then((comments) => {
+		console.log(comments);
+		const commentElement = document.body;
+		commentElement.innerHTML += 'Here are the comments: \n';
+        for (var i = 0; i < comments.length; i++) {
+            commentElement.innerHTML += comments[i] + '\n';
+        }
```

Comment:
nit - Reading over[ the style guide](https://google.github.io/styleguide/jsguide.html#features-for-loops) looks like `for-of` loops should be preferred (and are also a bit more readable).

Reply:
thanks! updated!